pdm/models/pip_shims.py

```diff
@@ -79,3 +80,10 @@ def get_package_finder(
 
     build_kwargs["ignore_requires_python"] = ignore_requires_python
     return install_cmd._build_package_finder(**build_kwargs)  # type: ignore
+
+
+unpack_url = (
+    partial(unpack_url, verbosity=0)
+    if "verbosity" in inspect.getfullargspec(unpack_url).args
+    else partial(unpack_url, verbosity=0)
```

Comment:
I think you meant

```suggestion
    unpack_url
```
?

Reply:
@pawamoy mypy fails if just `else unpack_url`

Reply:
`# type: ignore` it, or use a partial but without `verbosity=0` I guess?

Reply:
Ah, no, better: since there's only one thing to do, remove the ternary expression:

```python
if "verbosity" in inspect.getfullargspec(unpack_url).args:
    unpack_url = partial(unpack_url, verbosity=0)
```

Reply:
Is it right in the global scope?

Reply:
Sure, why not?

Reply:
Ok.